calculator/parser.py

```diff
@@ -139,3 +182,96 @@ def rpn(tokens):
                 ",".join(str(t) for t in tokens)
             )
         )
+
+
+def infix(token_list):
+
+    """
+    Parses Infix notation and returns the equivilant RPN form
+
+    Parameters
+    ==========
+    token_list : list
+        The list of infix tokens
+
+    Returns 
+    --------
+    output : list
+        This is the output queue where the final answer is arranged
+    
+    """
+
+    # Output Queue
+    output = []
+
+    # Operators Stack
+    operators = []
+
+    iteration_num = 1
+
+    # Iterating through each token in the token_list
+    for token in token_list:
+
+        # If the token is the a digit, or:
+        # 1. It has a negative sign in it and the whole number is not just a negative sign
+        # 2. or a decimal point is in the token
+        if (
+            str(token).isdigit()
```

Comment:
Can you just replace this with `is_numeric_token(token)` ?

Reply:
this is still pending

Reply:
👍